test/functional/feature_assumeutxo.py

```diff
@@ -216,20 +224,19 @@ def run_test(self):
         snapshot_prev_hash = n1.getblockhash(height=SNAPSHOT_BASE_HEIGHT-1)
         snapshot_hash = n1.getblockhash(height=SNAPSHOT_BASE_HEIGHT)
 
-        # nTx of the starting block should be 1 because it actually has one
-        # transaction. nTx of later blocks should be fake 1 values set by
-        # snapshot loading code.
+        # nTx of the starting block should be 1 because it has one transaction.
+        # nTx of later blocks should be unset.
         assert_equal(n1.getblockheader(start_hash)["nTx"], 1)
-        assert_equal(n1.getblockheader(start_next_hash)["nTx"], 1)
-        assert_equal(n1.getblockheader(snapshot_prev_hash)["nTx"], 1)
-        assert_equal(n1.getblockheader(snapshot_hash)["nTx"], 1)
+        assert_equal(n1.getblockheader(start_next_hash)["nTx"], 0)
+        assert_equal(n1.getblockheader(snapshot_prev_hash)["nTx"], 0)
+        assert_equal(n1.getblockheader(snapshot_hash)["nTx"], 0)
 
         # nChainTx of the starting block should be its height + 1. nChainTx of
         # the snapshot block should be snapshot_nchaintx. nChainTx of blocks in
-        # between these should be fake values set by snapshot loading code.
+        # between these should be unset.
         assert_equal(n1.getchaintxstats(nblocks=1, blockhash=start_hash)["txcount"], START_HEIGHT + 1)
-        assert_equal(n1.getchaintxstats(nblocks=1, blockhash=start_next_hash)["txcount"], START_HEIGHT + 2)
-        assert_equal(n1.getchaintxstats(nblocks=1, blockhash=snapshot_prev_hash)["txcount"], SNAPSHOT_BASE_HEIGHT)
+        assert_equal(n1.getchaintxstats(nblocks=1, blockhash=start_next_hash)["txcount"], 0)
```

Comment:
CI failed on this line: https://cirrus-ci.com/task/5549867297144832?logs=ci#L2880